bundles/org.openhab.automation.jsscripting/src/main/feature/feature.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+	Copyright (c) 2010-2020 Contributors to the openHAB project
+
+	See the NOTICE file(s) distributed with this work for additional
+	information.
+
+	This program and the accompanying materials are made available under the
+	terms of the Eclipse Public License 2.0 which is available at
+	http://www.eclipse.org/legal/epl-2.0
+
+	SPDX-License-Identifier: EPL-2.0
+-->
+
```

Comment:
This header already gets added when the features are aggregated. Probably we need to update this in the skeleton.

```suggestion
```

Reply:
We fixed that https://github.com/openhab/openhab-core/pull/2063 after you started this PR. :wink: